examples/quick-start/resnet50_libtorch/config.pbtxt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
```

Comment:
These additions and changes look great, fantastic work! Could you double check whether this is the right header? I think we now use "NVIDIA CORPORATION & AFFILIATES" on copyright headers.

Reply:
Done